### PR TITLE
Added initial snippets

### PR DESCRIPTION
**Pull request by ghost**
Added initial snippets for:
- Saga
- Testing Saga
- Console.log
- Common Imports
- Store Dispatch
- Arrow Function
- New react class
- Unroll, it, describe for jest
- `called`, `equal`, `falsy` and `truthy` for expect
- `router`, `route` and self closing `route` for react-router

### CODE REVIEW

**Comment by shishiranshuman** [comment]
Why is the PR against the `master`. Please create a version branch.

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+4. Select `Install Extension`
+5. Choose the extension
+6. Reload Visual Studio Code
+
+
+## Contribute
+
+This is an open source project and if you want to contribute, feel free to create issue in our [repository](https://github.com/causecode/snips-react) or create pull requests.
+
+
```
Comment (shishiranshuman) [inline README.md]: Add a block for the `License` and just display `MIT` in it's body

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+| Snippet                      | Purpose                                     |
+|------------------------------|---------------------------------------------|
+| `react`                      | Snippet for default import (React & Radium) |
+| `store`                      | Snippet for store.dispatch                  |
+| `saga`                       | Snippet for Saga (watcher & worker)         |
+| `arrow`                      | Snippet for Arrow Function                  |
+| `it`                         | Snippet for it (testing)                    |
+| `unroll`                     | Snippet for unroll (testing)                |
+| `describe`                   | Snippet for describe (testing)              |
```
Comment (shishiranshuman) [inline README.md]: Change the description to `Snippet for the Jest test suite.`

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
\ No newline at end of file
+# snips-react
+
+This VS Code extension adds snippets for React in your editor for typescript.
+
+### TypeScript React Snippets (.tsx)
+
+| Snippet                      | Purpose                                     |
+|------------------------------|---------------------------------------------|
+| `react`                      | Snippet for default import (React & Radium) |
```
Comment (shishiranshuman) [inline README.md:9]: Please add more snippets as discussed and accordingly change this snippet text. People might use the word `react` at other places as well.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "snips-react",
+    "displayName": "snips-react",
+    "version": "0.0.1",
+    "description": "A snippet pack to make you more productive working with React",
+    "publisher": "causecode",
+    "author": "CauseCode Technologies",
+    "license": "MIT",
+    "engines": {
+        "vscode": "^1.19.0"
```
Comment (shishiranshuman) [inline package.json]: Is caret really needed?

File: snippets/typescriptreact.json [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+            "}, [ // tslint:disable-next-line",
+            "\t[${0}],",
+            "\t[],",
+            "\t[],",
+            "]);\n"
+        ],
+        "description": "Snippet for unroll (testing)"
+    },
+
+    "it for jest": {
```
Comment (adityamedhe-cc) [inline snippets/typescriptreact.json]: `Test case for Jest`

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Done.

File: package.json (hunk 4 above)
Comment (ghost) [reply package.json]: Yes, it indicates compatibility with VS Code version `1.19.0` and onwards, and cannot be `*`. More details [here](https://code.visualstudio.com/docs/extensionAPI/extension-manifest).

File: snippets/typescriptreact.json (hunk 5 above)
Comment (ghost) [reply snippets/typescriptreact.json]: Done.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Done

**Comment by ghost** [comment]
Added new snippets and fixed suggested changes.

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md:9]: Fixed.

**Pull request merged**